docs/ABI.rst

```diff
@@ -1027,6 +1031,12 @@ Types
   METATYPE-REPR ::= 'T'                      // Thick metatype representation
   METATYPE-REPR ::= 'o'                      // ObjC metatype representation
 
+  SIL-BOX-FIELD-MUTABILITY ::= 'm'           // mutable field
+  SIL-BOX-FIELD-MUTABILITY ::= 'i'           // immutable field
+                                             // The number of mutability
+                                             // specifiers matches the length
+                                             // of the preceding type-list.
+
```

Comment:
Fair point. Like I mentioned to you in person, I just realized that I could encode the mutability of box fields by reusing the `inout` type list element encoding, which should be simpler to parse and more efficient.